caravel/migrations/versions/422116f431b7_add_d_timestamp_format_for_tables.py

```diff
@@ -0,0 +1,30 @@
+"""add d timestamp format for tables
+
+Revision ID: 422116f431b7
+Revises: 956a063c52b3
+Create Date: 2016-05-20 19:11:57.218062
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '422116f431b7'
+down_revision = '956a063c52b3'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+	try:
```

Comment:
![img](https://s-media-cache-ak0.pinimg.com/736x/09/14/2f/09142fb438441af92c9ea458657380d2.jpg)
failing would result in a broken app, let's fail early in the migration script if anywhere